test_nettoyage.py

```diff
@@ -0,0 +1,296 @@
+import pytest
+import json
+import os
+import pandas as pd
+import nettoyage
+import numpy as np
+from pandas.testing import assert_frame_equal
+
+# Partie recuperation de données.
+with open(os.path.join("confs", "config_data.json")) as f:
+    conf = json.load(f)
+path_to_data = conf["path_to_data"]
+decp_file_name = conf["decp_file_name"]
+with open(os.path.join(path_to_data, decp_file_name), encoding='utf-8') as json_data:
+    data = json.load(json_data)
+# On diminue la taille du fichier pour rendre le traitement plus rapide
+L_data = data["marches"][:5000]
+# On reforme la bonne structure
+data = {"marches": L_data}
```

Comment:
Si l'idée c'est de partager une donnée entre plusieurs tests, utiliser les [fixture pytest](https://docs.pytest.org/en/latest/how-to/fixtures.html), tu peux aussi trouver des examples d'utilisation dans les tests pypel

Reply:
Et si ce n'est pas partagé, le déplacer dans la fonction de test où c'est utilisé